### DISCUSSION TITLE
SB 5.1 - OCI - Enrollment data for a batch is not showing in course dashboard

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-Ed/Community#480] [Bugs] [answered]
Below are the steps followed

- Login as a content creator and visit a course page
- Go to the "View course dashboard" link
- Choose a batch
- The total count of enrollments shows no data

**Comment by ddevadat** [comment]
![image](https://user-images.githubusercontent.com/27432927/236993491-60f84428-c98f-4d63-8c3a-f98841985f6a.png)

**Comment by ddevadat** [comment]
we want to understand from where this data is fetched

**Comment by ddevadat** [comment]
@gandham-santhosh could you please help on this.
There was a similar issue https://github.com/project-sunbird/sunbird-community/discussions/52

as per that we checked the druid telemetry-rollup-syncts is running.
we couldnt find any druid ingestion flinkjob. Could you please let us know which job is this. We checked in the dp flink jobs. There
is a job name druid-validator flinkjob running

**Reply by ddevadat** [reply]
any update on this issue?

**Comment by ddevadat** [comment]
any update on this issue?

**Comment by chitranshu-keshav** [comment]
@rajeevsathish @vinukumar-vs Can you please get back on this

**Reply by gandham-santhosh** [reply]
@anandp504 can you please help here?

**Comment by ddevadat** [comment, marked as answer]
this issue is fixed. The issue the druid ip was pointing to raw cluster in lms config map. It should be pointing to rollup cluster

```
druid_proxy_api_host:
collection_summary_agg_data_source: audit-rollup-syncts
```

In other words, it should point to the ip where the druid datasource audit-rollup-syncts is present
